AcquisitionInfo/explainer.md

```diff
@@ -265,6 +265,34 @@ the acquisition information that is recorded for the originally acquired applica
 other installations. This applies to any and all other installation for the same user profile across any other device supported
 by the UA.
 
+For example, a user may discover App A through an ad campaign run on the Microsoft Store. The user proceeds to install App A on
+Device A. Running `getDetails()` from the newly acquired app would return the following:
+
+```js
+details = {
+  installSource: "apps.microsoft.com",
+  attributionId: "adCampaign",
+  ... // Additional attribution information
+} 
+```
+
+The same user logs in on Device B and logs into the profile that installed App A on Device A. Sync would install App A once again
+but on Device B, at which point the `getDetails()` payload would return the following:
+
+```js
+details = {
+  installSource: "Sync",
```

Comment:
Wasn't the `installSource` supposed to remain the same with the addition of `sync`?
```suggestion
  installSource: "apps.microsoft.com",
```